src/charts/index.js

```diff
@@ -3,8 +3,10 @@ export {default as Donut} from './donut/Donut.js';
 export {default as GroupedBar} from './groupedBar/GroupedBar.js';
 export {default as Legend} from './legend/Legend.js';
 export {default as Line} from './line/Line.js';
+export {default as ResponsiveContainer } from './helpers/responsiveContainer.js';
 export {default as Sparkline} from './sparkline/Sparkline';
 export {default as StackedArea} from './stackedArea/StackedArea.js';
 export {default as StackedBar} from './stackedBar/StackedBar.js';
 export {default as Step} from './step/Step.js';
 export {default as Tooltip} from './tooltip/Tooltip.js';
+export {default as withResponsiveness} from './helpers/withResponsiveness.js';
```

Comment:
Let's put them together and add a space between them and the chart components, just for differentiate them in the code.